apps/ledger-live-desktop/src/renderer/components/Onboarding/Screens/SelectDevice/index.tsx

```diff
@@ -34,9 +34,14 @@ export function SelectDevice() {
 
   const handleDeviceSelect = useCallback(
     (deviceModelId: DeviceModelId) => {
-      track("Onboarding Device - Selection", { deviceModelId });
-      setDeviceModelId(deviceModelId);
-      history.push("/onboarding/select-use-case");
+      // TODO: use a feature flag to do this properly
+      if (deviceModelId === "nanoFTS") {
+        history.push("/sync-onboarding");
```

Comment:
Maybe add `track("Onboarding Device - Selection", { deviceModelId });` later to track FTS